components/frontend/src/header_footer/Menubar.test.js

```diff
@@ -90,7 +90,8 @@ it("logs out", async () => {
     auth.logout = jest.fn().mockResolvedValue({ ok: true })
     const set_user = jest.fn()
     renderMenubar({ set_user: set_user, user: "jadoe" })
-    await fireEvent.click(screen.getByText(/Logout/))
+    await fireEvent.click(screen.getByRole("button", { name: "Logout" }))
+    await fireEvent.click(screen.getByRole("menuitem", { name: "Logout" }))
```

Comment:
Two clicks?

Reply:
Yeah, the first one the button with the user avatar to open the menu and then on the menu item "Logout" to actually log out. I guess it's surprising because the user avatar button has "Logout" as accessible name. I'll change that to something like "User options".